validator/testcases/content.py

```diff
@@ -297,13 +298,13 @@ def _process_file(err, xpi_package, name, file_data, name_lower,
         try:
             sub_xpi = XPIManager(package, mode="r", name=name,
                                  subpackage=is_subpackage)
-        except Exception:
+        except BadZipfile:
```

Comment:
Do we have a good reason to have two different places where we try to open the zipfile, and raise an error if we can't?